### PR TITLE
Add console documentation

### PR DESCRIPTION
**Pull request by Yacoubou-seidou**
### Add console documentation
- To get all console
- To get a single console
- To create a console
- To delete a console
Notice documentation is made for methods that existed in the controller API for the console.

### CODE REVIEW

File: swagger/v1/swagger.yaml [hunk 1]
```diff
@@ -4,6 +4,176 @@ info:
...
+                    - id
+                    - name
+                    - photo
+                    - purchase_price
+                    - rental_price
+                    - description
+                required:
+                - status
+                - message
+                - data
```
Comment (stephenkati) [inline swagger/v1/swagger.yaml]: Please add tests for creating and deleting a post.

Here is the breakdown of the issue:

After pulling your changes this is what I see
Screenshot from 2023-08-18 03-21-51

When I run `rake rswag:specs:swaggerize`, the documentation changed to:
Screenshot from 2023-08-18 03-21-09

Please add integration tests for the `post` and `delete`

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '>=3.1.2'
+ruby '3.2.2'
```
Comment (stephenkati) [inline Gemfile]: Please change back ruby version to `ruby '>=3.1.2'` in order to allow seamless integration for every collaborator

**Review by stephenkati — CHANGES_REQUESTED** [review]
Hi @Yacoubou-seidou , @Timbar09 

### REQUESTED cHANGES :recycle: 

**Highlights**
- Linters :100:
- Workflow :heavy_check_mark: 
- Summary :heavy_check_mark: 
#
Good Job so far
Implement requested changes and request for a review

File: Gemfile (hunk 2 above)
Comment (Timbar09) [reply Gemfile]: Great, understood, @stephenkati. 👍

Please make sure you do the same. I had to change the version as well when I pulled changes from you.

**Comment by Timbar09** [comment]
Thanks for the feedback, @stephenkati. I see the issue. I'm on it.

**Review by stephenkati — APPROVED** [review]
Hi @Yacoubou-seidou @Timbar09 

### STATUS APPROVED :green_circle: 

**Highlights**
- Linters:100:
- All methods documentated:heavy_check_mark: 
- Summary:heavy_check_mark: 

Nothing else to add, proceed to merge :shipit:

**Comment by Timbar09** [comment]
Awesome. 🙃

**Pull request merged**